services/cron/tasks_extended.go

```diff
@@ -158,7 +157,7 @@ func registerUpdateGiteaChecker() {
 	})
 }
 
-func registerDeleteOldSystemNotices() {
+func registerDeleteOldSystemNotices(ctx context.Context) {
```

Comment:
This should not be needed and/or wrong. The cron has it's own context (line 168).

Reply:
It uses the cron context from line 168

Reply:
Still, the method parameter is useless and can thus be removed.